Makefile

```diff
@@ -1,42 +1,46 @@
-.PHONY: all all-local build build-local check check-code check-format run-tests check-local integration-tests install-deps coveralls circle-ci start-services clean
+.PHONY: all all-local build build-local clean cross cross-local check check-code check-format run-tests integration-tests check-local coveralls circle-ci-cross circle-ci-build circle-ci-check circle-ci
 SHELL=/bin/bash
-build_image=libnetworkbuild
+build_image=libnetwork-build
```

Comment:
I am not sure why #667 removed the dash from the name.
But you may want to check with @LK4D4 before re-adding the `-`.